### PR TITLE
Always test deployments using forks + separate unit-tests and deployment testing during CI

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,14 +63,10 @@ This suite is called “nightly” since it’s scheduled to run every day at mi
 
 You can run deployment unit-tests (which are also part of the full test suite) via:
 
-```sh
-yarn test:deploy
-```
-
-You can also run a specialized set of deployment tests against a mainnet fork via:
+You can test new deployments (and the health of the network) against a mainnet fork via:
```
Comment (yudilevi) [inline README.md:66]: need to delete the previous line?

File: deploy/tests/000024-enable-beta-pools.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { NATIVE_TOKEN_ADDRESS } from '../../utils/TokenData';
+import { toCents, toWei } from '../../utils/Types';
+import { expect } from 'chai';
+import { getNamedAccounts } from 'hardhat';
+
+describeDeployment(__filename, () => {
+    let networkSettings: NetworkSettingsV1;
+    let poolCollection: PoolCollection;
+
+    // TODO: make sure to update the limits and the rates before running the script in production
```
Comment (yudilevi) [inline deploy/tests/000024-enable-beta-pools.ts:14]: can probably remove this comment now (and similar comments)

File: deploy/tests/000025-create-beta-rewards-programs.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import { BNT } from '../../components/LegacyContracts';
+import { describeDeployment } from '../../test/helpers/Deploy';
+import { DeployedContracts } from '../../utils/Deploy';
+import { duration } from '../../utils/Time';
+import { NATIVE_TOKEN_ADDRESS } from '../../utils/TokenData';
+import { toWei } from '../../utils/Types';
+import { expect } from 'chai';
+import { getNamedAccounts } from 'hardhat';
+
+// TODO: make sure to update the starting time of all beta programs
```
Comment (yudilevi) [inline deploy/tests/000025-create-beta-rewards-programs.ts:11]: can probably remove this comment now

File: deploy/tests/000024-enable-beta-pools.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            const tokenPriceInCents = BETA_TOKEN_PRICES_IN_CENTS[tokenSymbol as BetaTokens];
+            const depositLimit = toWei(TKN_DEPOSIT_LIMIT_IN_CENTS).div(tokenPriceInCents);
+
+            const data = await poolCollection.poolData(address);
+            expect(data.depositingEnabled).to.be.true;
+            expect(data.depositLimit).to.equal(depositLimit);
+
+            const bntVirtualBalance = tokenPriceInCents;
+            const tokenVirtualBalance = BNT_TOKEN_PRICE_IN_CENTS;
+            const initialDeposit = minLiquidityForTrading.mul(tokenVirtualBalance).div(bntVirtualBalance).mul(3);
```
Comment (barakman) [inline deploy/tests/000024-enable-beta-pools.ts:59]: A comment on where this '3' stems from might be useful.

**Review by barakman — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lbeder) [reply README.md:66]: Oops

File: deploy/tests/000024-enable-beta-pools.ts (hunk 2 above)
Comment (lbeder) [reply deploy/tests/000024-enable-beta-pools.ts:14]: 👍

File: deploy/tests/000025-create-beta-rewards-programs.ts (hunk 3 above)
Comment (lbeder) [reply deploy/tests/000025-create-beta-rewards-programs.ts:11]: 👍

**Review by yudilevi — APPROVED** [review]

**Pull request merged**